azure-gaming-pr/reference-architectures/cognitive.md

```diff
@@ -20,10 +20,13 @@ Cognitive Services can be used to improve a variety of gaming scenarios, both in
 * [Text Translation](./cognitive-text-translation.md)
   * Translate text between languages so players across the world can communicate with each other in their native language.
 * [Text to Speech](./cognitive-text-to-speech.md)
-  * Help being everyone into the conversation by translating text to audio.
+  * Help bring everyone into the conversation by synthesizing text to audio.  
+  * Game dialogue prototyping to shorten the amount of time and money spent in product to get the game to market sooner, you can rapidly swap lines of dialog and listen to variations in real-time to ensure accuracy. 
+  * Make gaming accessible and localized to read out the storylines or instructions to individuals unable to read the text or language, or younger users that have not yet to be taught to read. 
```

Comment:
Make the game more accessible. Players can listen to storylines and instructions in their preferred language. This enables players, including young players who haven't learnt to read and write, who are unable to read or read text in a particular language to be able to play your game.

Reply:
> 

Updated. Thanks a lot.